charts/prism/values.yaml

```
@@ -682,3 +697,62 @@ deployments:
+    replicaCount: 1
+    metrics:
+      enabled: true
+      serviceMonitor:
+        enabled: false
+    namespace: mg
+  crds:
+    install: true
+  dashboard:
+    enabled: true
+  nameOverride: argo-rollouts
+  fullnameOverride: argo-rollouts
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)